_posts/2017-08-21-spatial-data-10-4.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "WDC Spatial Data in Tableau 10.2"
```

Comment:
Ahh yes, copy paste bug.  Thanks for catching!  If you wouldn't mind updating that in your pass through I would appreciate it.